Aptoide/aptoide-client-v8
# [feature/ASV-1393] Similar APPC apps

**What does this PR do?**

   Adds a new bundle in the similar apps section of app view. This bundle is essentially similar apps but for AppCoins campaigns (with an header fading animation). As a result, the similar app section was refactored to function similarly to home bundles. The way we move the similar apps up and down in appview was also refactored: instead of having two similar apps section and hiding / showing, we only have 1 section where we use placeholders to properly move the view (by adding/removing it).

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] cm.aptoide.pt.app.view.similar package

**How should this be manually tested?**

  Two things that need to be tested:
  1) Check if the layout rules are correct. If the app is an appcoin app, similar appc bundle should appear first and then regular similar apps. If the app is not an appcoin app, the reverse should happen.
  2) Check if MoPub native ads still work
  
   And generally check if the rest is ok, e.g. similar appc apps being fetched correctly, if there's nothing broken in similar apps....

**What are the relevant tickets?**

  [ASV-1393](https://aptoide.atlassian.net/browse/ASV-1393)

**Code Review Checklist**
